src/extension.ts

```diff
@@ -28,9 +28,9 @@ export function activate(context: ExtensionContext) {
   const logger = getLogger().getChildLogger({ label: "activate" });
   logger.info("The App-Studio-Toolkit Extension is active.");
 
-  const workspaceAPI: BasWorkspaceApi = getBasWorkspaceAPI(WorkspaceImpl);
-  void initTagsContexts(workspaceAPI);
-  void initProjectTypeWatchers(workspaceAPI);
+  const workspaceAPI: BasWorkspaceApi = getBasWorkspaceAPI();
+  void initTagsContexts(workspaceAPI as WorkspaceApi);
+  void initProjectTypeWatchers(workspaceAPI as WorkspaceApi);
```

Comment:
There is a bit of confusion here between the internal (implementation) and external (api) concerns.
- Internally we may need the full `WorkspaceApi` instance and all its methods.
- Externally we are limiting the available APIs.

These two `initXYZ` functions are internal, so why are they working with the **External** proxy?
Also note we don't really need to save a reference to the proxied instance.